### PR TITLE
Changes made in order to automatically generate a test report in chrome

### PR DESCRIPTION
**Pull request by DanDaika**

### CODE REVIEW

File: conf.js [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
   specs: ['./tests/tests.js'],
   multiCapabilities: [{
-    browserName: 'firefox'
+    browserName: 'chrome'
   }],
-  jasmineNodeOpts: {defaultTimeoutInterval: 60000}
+  jasmineNodeOpts: {defaultTimeoutInterval: 60000},
+  onPrepare: () => {
+    jasmine.getEnv().addReporter(new HtmlReporter({
+      baseDirectory: 'reports/'+timeStamp,
```
Comment (cspireasv) [inline conf.js:33]: The reports folder structure doesn't comply with the required specification.
image

Comment (DanDaika) [reply conf.js:33]: The timestamp was updated to the desired format.

**Comment by cspireasv** [comment]
Looks good.

**Review by cspireasv — COMMENTED** [review]
Looks good

**Review by cspireasv — APPROVED** [review]

**Pull request merged**